src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -48,6 +48,7 @@ public class Navigator : Control
         [EditorBrowsable(EditorBrowsableState.Never)]
         public Navigator() : base()
         {
+            Layout = new AbsoluteLayout();
```

Comment:
as @hyunjushin has commented, please resolve the build warnings shown line 386, 389, and so on.

Reply:
The CA2000 in Navigator is false alarm.
To remove the CA2000 false alarm in Navigator, please review the following PR.
https://github.com/Samsung/TizenFX/pull/2729

Reply:
OK. good! I got it